src/wrappers.ts

```diff
@@ -78,4 +78,9 @@ export class VSCode {
     public createOutputChannel(name: string) {
         return vscode.window.createOutputChannel(name);
     }
+
+    public runTask(name: string)
```

Comment:
What is this for?

Reply:
Was used in the first versions of the PR, will remove